src/pages/Category/ListTitle/ListTitle.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import './ListTitle.scss';
+
+class ListTitle extends Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedCategory: '',
+      selectedSubcategory: '',
+    };
+  }
+
+  selectedCategoty = (subtitleElement) => {
+    this.setState({ selectedCategory: subtitleElement.subTitle });
+  };
+
+  selectedSubcategory = (susubTitleElement) => {
+    this.setState({ selectedSubcategory: susubTitleElement.susubTitle });
+  };
```

Comment:
기능이 같으므로 하나의 함수로 합칠 수 있겠습니다 ;) 

```suggestion
onClick={() => this.selectedCategoty(e, el)}     // 2개의 인자를 전달받는 함수로 수정하고 함수를 아래처럼 하나로 합쳐주기



  selectedCategory = (e, el) => {
    let target = e.target.className === "subTitleList" ? subTitle : susubTitle;
    this.setState({ [el]: el.target });
  };
```

Reply:
결재완료